### PR TITLE
[3.x] Expose autotile_coord parameter in `TileMap.set_cellv`

### PR DESCRIPTION
**Pull request by Rubonnek**
This change is only relevant to the `3.x` branch given the `TileMap` rework on `master`.

### CODE REVIEW

File: doc/classes/TileMap.xml [hunk 1]
```diff
@@ -154,6 +154,7 @@
 			<argument index="2" name="flip_x" type="bool" default="false" />
 			<argument index="3" name="flip_y" type="bool" default="false" />
 			<argument index="4" name="transpose" type="bool" default="false" />
+			<argument index="5" name="autotile_coord" type="Vector2" default="Vector2( 0, 0 )" />
```
Comment (akien-mga) [inline doc/classes/TileMap.xml:157]: Please adjust the description too to include references to autotile coordinates like done for `set_cell`.

Comment (Rubonnek) [reply doc/classes/TileMap.xml:157]: Description updated.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.3.

**Pull request merged**